packages/teleterm/src/ui/services/workspacesService/workspacesService.ts

```diff
@@ -137,6 +137,12 @@ export class WorkspacesService extends ImmutableStore<WorkspacesState> {
       });
     };
 
+    // empty cluster URI - no cluster selected
+    if (!clusterUri) {
+      setWorkspace();
```

Comment:
Why do we need to call `setWorkspace` if we know that there's no `clusterUri`? Just to set `rootClusterUri` to `undefined`?

If so then I think we should just do it here, that would be more readable. Right now when reading this function you have to jump up to see what `setWorkspace` does, only to realize that you can actually ignore almost all of its body except one line.

Reply:
I agree, it is more readable now.